Add 2023-09-27 agenda and minutes (#573)
solid/specification: meetings/2023-09-27.md

### Replace panels with breakout task forces
URL: https://github.com/solid/process/issues/324

* SC: Processed a lot of panel repositories to reflect meetings. Some details like moving work items that are part of the repository to their own under solid org. Most of the work items are already in their own repositories. Solid process needs to be updated. CG should help clarify parts that are not CG/WG related in the process. Task force info we can sort out in specification repo.
* eP: Okay with closing the issue, we have a special meeting scheduled for Oct 10th. Maybe we can create a new tracking issue? If a work item does not have a dedicated repo we need to do that and archive repos/matrix channels. We need an issue to track.
* SC: Okay to keep open too. In the process repo I interpret as updating the process. If the issue is about how to transition from panels to task forces then an issue could be opened in the specification repo, where we can detail the expectations for a task force. We can document W3C guidelines in contributing guide. 

ACTION: SC to create discussion to improve contributing guide to detail how task forces are formed (in similar way to new work item proposal)

> * eP: We need to take into account panel meetings and ecosystem monitor. We need to plan if we want to merge all meetings into one, or archive.. we should discuss in the special meeting on Oct 10th.
> * SC: Also minutes up to a certain point are using another format before we had the template. To actually use in the ecosystem monitor we need to clean that up. So it'll be great to have the data for it requires some work. Panels will take a lot more energy to clean up minutes.
> * RG: During charter preparation I pointed out.. there are a lot of issues that are out of the charter scope. There needs to be a process to transition the larger issues. 
> * SC: Any objection to closing this issue and follow up with a discussion / PR? 

Reviewer: ```suggestion
* eP: We need to take into account panel meetings and ecosystem monitor. We need to plan if we want to merge all meetings into one, or archive... We should discuss in the special meeting on Oct 10th.
* SC: Also, minutes up to a certain point are using another format, from before we had the template. To actually use these in the ecosystem monitor, we need to clean that up. It'll be great to have the data for it, but it requires some work. Cleaning up minutes for panels will take a lot more energy.
* RG: During charter preparation, I pointed out... there are a lot of issues that are out of scope per charter. There needs to be a process to transition the larger issues. 
* SC: Any objection to closing this issue and following up with a discussion / PR? 
```

No objections.

* eP: After you create it, can you link to this one? 
* SC: Yes. 

### Solid Protocol Version 0.11.0
URL: https://github.com/solid/specification/milestone/7

* SC: Let's make sure to add any missing issues/PRs that can reasonably make it into this release. The ED includes class 3 and higher changes, and some in the pipeline. See [Solid Protocol ED Changelog](https://solidproject.org/ED/protocol#changelog).
* SC: See also discussion threads in https://matrix.to/#/!QxZtVBYQfMeMTnespj:gitter.im/$KJ7dUsbNFznsJ-M-xZKuBENuo698qCuPSuHz5bt2WZ4 and https://matrix.to/#/!QxZtVBYQfMeMTnespj:gitter.im/$BCTfbI-dh95G-9gYZW9LTmRsoQ064Yg3zmySf9rDSXU .
* SC: ED represents consensus already reached on those changes, so it's sensible that work continues from there. 
* eP: We'll know about final decision on WG in 2 weeks. Might be safer if everything is ready to make a PR to cut a release, once WG is created we can get approval from WG and they can pick up 0.11. 

> * MdJ: It'd be easier from testing perspective.. protocol now is at CG so if we tag at handover point that'd make it easier. Maybe WG will decide not to produce intermediate versions and then we're stuck with this for two years, and I'd like to be able to point to this tag. 

Reviewer: ```suggestion
* MdJ: It'd be easier from a testing perspective. Protocol is now at CG, so if we tag at handover point, that'd make it easier. Maybe WG will decide not to produce intermediate versions, and then we're stuck with this for two years. I'd like to be able to point to this tag. 
```

* SC: Emphasis on we don't know if/when WG will be approved. We don't need to rush. Cutting release is just making some changes which are mostly in place. I'd suggest focusing on clearing the milestones, issues that would belong in 0.11, focus on that and if that looks good we can make a PR and give it some time. Whoever wants to merge at that point can. 
* MdJ: If we don't reach the milestones for 0.11, then we can just tag 0.10.1 and we still have a tag to point to. 
* SC: WG can pick up a release document or ED and turn it into FPWD. This is a minor concern IMO.
* eP: Only risk I see is releases diverge and it's confusing.

> * MdJ: Agree. If we can and nobody objects we can decide on OCt 6th we either do 0.11 or 0.10.1. 

Reviewer: ```suggestion
* MdJ: Agree. If we can and nobody objects, we can decide on Oct 6 to do either 0.11 or 0.10.1. 
```

* eP: +1
* SC: Our job is to hand over latest consensus. WG will pick up most accurate useful version. For example, it does not make sense for WG to pick up 0.10 and re-do the changes we have done since in ED.

### Chat Client-Client spec new work item
URL: https://github.com/solid/specification/issues/553

* SC: Proposed by TBL.

> * SC: There were a few questions/considerations raised. If we can clarify that's great. I don't think there are any objections to this work item. TBL/Hadrian, would you like to comment on anything?
> * HZ: What needs to be decided? What kind of comment is expected? TBL created issue, I supported. There's a draft for the spec for August. What do we need to close issue and move forward? You indicated a work item has to be created... what else.

Reviewer: ```suggestion
* SC: There were a few questions/considerations raised. If we can clarify, that's great. I don't think there are any objections to this work item. TBL/Hadrian, would you like to comment on anything?
* HZ: What needs to be decided? What kind of comment is expected? TBL created issue; I supported. There's a draft for the spec for August. What do we need to close issue and move forward? You indicated a work item has to be created... what else?
```

* SC: I had a response in https://github.com/solid/specification/issues/553#issuecomment-1737144192

> * TBL: The issue has been overtaken by ??? there was a repo. In august I had time and wrote an initial draft which should have all the pieces in it. Whether or not the CG decides to form a ??? solid chat is part of SolidOS code and ecosystem for a long time. From the point of view of the spec we should get people to review. Next thing to do is the shapes. I'd like to use shapes to validate. If it gets adopted as work item, what effect does it have?
> * eP: My understanding is that it's already pretty much a work item. It already lists TimBL as editor and Virginia, Jeff and Timea, Hadrian should be added too if he's interested in being an editor. 

Reviewer: ```suggestion
* TBL: The issue has been overtaken by ??? There was a repo. In August, I had time and wrote an initial draft which should have all the pieces in it. Whether or not the CG decides to form a ???, Solid Chat has been part of SolidOS code and ecosystem for a long time. From the point of view of the spec, we should get people to review. Next thing to do is the shapes. I'd like to use shapes to validate. If it gets adopted as work item, what effect does it have?
* eP: My understanding is that it's already pretty much a work item. It already lists TimBL as editor and Virginia, Jeff, and Timea; Hadrian should be added, too, if he's interested in being an editor. 
```

* SC: We should focus on procedure and not contents of the work item. 
* VB: Wanted to clarify that I'm not a contributor. 
* TBL: Just a copy issue.
* SC: TBL is editor, contributors will be updated. Would be great to have another person to support more directly. When Hadrian is interested, does that entail authoring/editing? 
* HZ: Yes.
* SC: Which one? 
* TBL: What's the difference? 

> * SC: Generally speaking, if the concepts of the work are developed by Hadrian he'd be authoring, the editor is responsible that the information is coherent. Basic owner of the document. Authors might not necessarily have the best way to express requirements, structures.. editor will make the final call. 

Reviewer: ```suggestion
* SC: Generally speaking, if the concepts of the work are developed by Hadrian, he'd be authoring. The editor is responsible to see that the document is coherent. Basic owner of the document. Authors might not necessarily have the best way to express requirements, structures, etc. Editor will make the final call. 
```

* TBL: Does W3C have those concepts defined somewhere?
* SC: Yes. Some threads on expectations for an editor. Some specs with only editors means they are also the authors. 
* TBL: I don't think we should ask contributors to decide which one they want to be. 

> * SC: It's a rough expectation. If you want to change role, that's fine. It's about expecting Hadrian will be submitting new content or design of the chat spec or more on editorial. 
> * eP: Agree distinction is confusing and should be better documented. In most cases editors are also authors. Usually few more authors than editors. Usually editors are also authors. 

Reviewer: ```suggestion
* SC: It's a rough expectation. If you want to change role, that's fine. It's about whether to expect Hadrian will be submitting new content or design of the chat spec or more on editing content from others. 
* eP: Agree, distinction is confusing and should be better documented. In most cases, editors are also authors. Usually few more authors than editors.
```

* RG: This distinction is clearer ??? Chair can appoint editors when ??? 
* SC: That's in W3C process.
* RG: Some notion of editor in the process: https://www.w3.org/2021/Process-20211102/
* eP: Let's follow up in the issue: https://github.com/solid/chat/issues/2
* SC: Maybe TBL and Hadrian can follow up and update. 
* SC: What it entails to be in the CG: is this specification being worked on indepdently? Copyright is Solid org, in the charter we expect that to be W3C CG CLA and so forth. 
* TBL: CLA is not an org, it's an agreement.
* SC: W3C or solid org?
* TBL: Solid org. 
* SC: How can we have it as a CG work item if it's not following the charter? 
* TBL: The charter discussion happened after the chat spec was written. There was a compromise between W3C license and MIT and the compromise all code snippets should be public domain. 

> * VB: In WebID profile we have contributor role. 
> * SC: That said, in new work item proposal we said we'd have one owner and one other person to work through the work item. 
> * eP: Can we add a todo to resolve author/editors/contributors? And add an issue to chat spec to clear up copyright. 
> * TBL: Is it necessary for a work item to be adopted that there's content in the document? 

Reviewer: ```suggestion
* VB: In WebID Profile, we have contributor role. 
* SC: That said, in new work item proposal, we said we'd have one owner and one other person to work through the work item. 
* eP: Can we add a todo to resolve author/editors/contributors? And add an issue to chat spec to clear up copyright?
* TBL: Is it necessary that there be content in the document for a work item to be adopted?
```

* SC: There's no blockers/barriers to it begin a work item. Just clarification on contributors. 
* SC: https://github.com/solid/chat/issues/2
* SC: We have an issue and a sense of who's contributing. Copyright line should be updated to match what's expected in the charter. Solid org > W3C. 

ACTION: TBL to nudge HZ about role. TBL (or SC?) to PR resolving issue 2.

ACTION: SC to PR copyright change.

ACTION: SC to add a reference to chat spec from TR page (with the understanding that copyright and contributors will be updated)

* TBL: I want to also add a list of implementations. 

> * SC: Agree. Overlaps with Solid QA work. Besides listing, if it's accompanied by a test we need to provide test results. Specs do link out to a document that contains test reports. 

Reviewer: ```suggestion
* SC: Agree. Overlaps with Solid QA work. Besides listing, if it's accompanied by a test, we need to provide test results. Specs do link out to a document that contains test reports. 
```
